src/action.ts

```diff
@@ -112,13 +112,16 @@ async function action(pr: PullRequest): Promise<void> {
     JSON.parse(getInput('pipeline_settings'))
   );
 
+  const ref = getInput('git_ref').replace('refs/heads/', '') || 'master';
+  debug(`Using base ref: '${ref}'`);
```

Comment:
It is worth checking how this is expanded in combination with `git_url`, specifically, is the `${{ github.base_ref }}` expanding to the `ref` of the PR (`refs/pull/123/head`) or that of the fork (`merging_branch`).

Reply:
git_url is used for Testing Farm repos data. Not to git repo where the code is testing.

Reply:
@phracek these are about setting default options for projects where tmt tests are included upstream. The user would still be able to override these, and this was partially addressed in #189. Note my other comment [^1] for why it may be beneficial to move the defaults in-code.

Meta issue: https://github.com/sclorg/testing-farm-as-github-action/issues/185

[^1]: https://github.com/sclorg/testing-farm-as-github-action/pull/190#discussion_r1618707655